go/vt/vtctl/workflow/server.go

```diff
@@ -1720,6 +1720,12 @@ func (s *Server) VDiffCreate(ctx context.Context, req *vtctldatapb.VDiffCreateRe
 		tabletTypesStr = discovery.InOrderHint + tabletTypesStr
 	}
 
+	// This is a pointer so there's no ZeroValue in the message
+	// and an older v18 client will not provide it.
+	if req.MaxDiffDuration == nil {
+		req.MaxDiffDuration = &vttimepb.Duration{}
+	}
```

Comment:
Codecov says there is no code coverage for this if condition but i am guessing it is tested in E2E tests?

Reply:
Yeah, it would be covered there. I believe the `codecov/patch` test is pointing out that this patch/PR ADDs this line, but it does NOT ADD a test which covers it.

Reply:
I'll actually create one quickly...